src/site/stages/build/plugins/create-resources-and-support-section.js

```diff
@@ -47,56 +47,76 @@ function groupByTags(allArticles) {
       },
     } = article;
 
-    fieldTopics.map(fieldTopic => fieldTopic.entity.name).forEach(tag => {
-      const taggedArticles = tagMap[tag] || [];
-      tagMap[tag] = taggedArticles.concat(article);
+    fieldTopics.map(fieldTopic => fieldTopic.entity).forEach(fieldTopic => {
+      const articleListing = articleListingsByTag[fieldTopic.name];
+
+      if (!articleListing) {
+        articleListingsByTag[fieldTopic.name] = {
+          articles: [article],
+          entityUrl: fieldTopic.entityUrl,
+        };
+      } else {
+        articleListing.articles.push(article);
+      }
     });
   }
 
-  return tagMap;
+  return articleListingsByTag;
 }
 
 function groupByCategory(allArticles) {
-  const categoryMap = {};
+  const articleListingsByCategory = {};
   const categorizedArticles = allArticles.filter(a => !!a.fieldPrimaryCategory);
 
   for (const article of categorizedArticles) {
-    const {
-      fieldPrimaryCategory: {
-        entity: { name: primaryCategoryName },
-      },
-      fieldOtherCategories,
-    } = article;
-
-    const primaryCategoryArticles = categoryMap[primaryCategoryName] || [];
-    categoryMap[primaryCategoryName] = primaryCategoryArticles.concat(article);
+    const { fieldPrimaryCategory, fieldOtherCategories } = article;
+
+    const primaryCategoryName = fieldPrimaryCategory.entity.name;
+    const primaryCategoryArticleListing =
+      articleListingsByCategory[primaryCategoryName];
+
+    if (!primaryCategoryArticleListing) {
+      articleListingsByCategory[primaryCategoryName] = {
+        articles: [article],
+        entityUrl: fieldPrimaryCategory.entity.entityUrl,
+      };
```

Comment:
This is where the Drupal "entityUrl" is taken into account during the build for article listings by category